packages/core/src/components/pagination/pagination.tsx

```diff
@@ -1,13 +1,28 @@
-import { h, Component, ComponentInterface, Event, EventEmitter, Prop } from '@stencil/core';
+import { h, Component, ComponentInterface, Element, Event, EventEmitter, Fragment, Prop, State } from '@stencil/core';
 
 import { PaginationSelectPageEvent } from './pagination.interfaces';
 
+export type LabelSizeMap = { [key in string]: number; }
+
 @Component({
   tag: 'dso-pagination',
   styleUrl: 'pagination.scss',
   shadow: true,
 })
 export class Pagination implements ComponentInterface {
+  private sizePositionsMap: LabelSizeMap = {
+    small: 4,
+    medium: 8,
+    large: 10,
+  }
+
+  private resizeObserver?: ResizeObserver;
+
+  @Element()
+  host!: HTMLElement;
+
+  @State()
+  elementPositions!: number
```

Comment:
`availablePositions` lijkt me passender. Ik denk ook dat deze property beter optioneel kan zijn. (Gezien dat .getSize() een Promise is.

Reply:
eens